src/script/common/c_content.cpp

```diff
@@ -1516,6 +1516,17 @@ void push_flags_string(lua_State *L, FlagDesc *flagdesc, u32 flags, u32 flagmask
 	lua_pushlstring(L, flagstring.c_str(), flagstring.size());
 }
 
+void push_flags_table(lua_State *L, FlagDesc *flagdesc, u32 flags, u32 flagmask)
```

Comment:
4 bytes lost each. CSM restricitons are `u64`.

Reply:
I'm aware but for consistency I made this one use `u32` too.
Perhaps get_csm_restrictions should just reimplement this.